README.rst

```diff
@@ -30,7 +31,7 @@ in Thoth's middletier namespace.
 
   * graph-refresh job is run in Thoth's frontend namespace
   * templates for solvers are picked from Thoth's infra namespace - see installed solvers endpoint on Management API for available solvers in the deployment
```

Comment:
Are there templates also for package-analyzer?

Reply:
If this https://github.com/thoth-station/package-analyzer/blob/master/openshift/job-template.yaml what you mean by templates, then I think yes.

Reply:
Then we should modify also the line related to templates WDYT? :)